### DISCUSSION TITLE
Simplify project list

### DISCUSSION THREAD

**Discussion by jens-br** [nest-desktop/nest-desktop#406] [Ideas]
As discussed in our virtual meeting, it seems to be reasonable to simplify the project list (see below): With that, we could group the functions for all projects into a button panel above the search field - and remove the "Rename" field, which could be integrated into the button with three dots (if we want to move the functions for single projects into a context menu).

 * We discussed if the icon with the three dots should be changed to a pencil, but that would only indicate "Editing"/"Renaming", and the other functions like reloading would be not clear for the user.
 * We will see if we have enough space to keep the three dots for the selected project visible for the whole time, but we have to keep in mind that there is not so much space, since the icon for spatial/other activity needs also place, 

- [x] The save icon (for saving the changes made to the currently selected project) should be integrated into the projects list: The currently selected project will contain the https://icon-sets.iconify.design/mdi/content-save-check-outline/ icon (in grey) if everything is saved, but the https://icon-sets.iconify.design/mdi/content-save-alert-outline/ icon instead in case there are unsaved changes.
- [ ] This would pose the opportunity to remove or restructure the system bar: We will discuss whether we rather integrate the status icons for the backends in the area next to the simulation button - or if we just keep it, since the reduced design looks quite nice as an environment for the status icons. However, this is still a subject of discussion.
- [x] The icon bar and the search bar should be always visible - even when the user scrolls in the projects list.
- [ ] Undo/Redo operations should work with this new feature (e.g. when a project is saved and the user reverts the changes, the save-warning icon should be visible).
- [ ] The PROJECTS menu in the system bar should be deactivated for now (and not be rendered), but the code will be kept, since it might happen in the future that the toolbar has too many icons to display, so that we would need that menu again.
- [x] The model list should be restructured in a similar way.
- [ ] The width icons in the toolbar should grow/shrink flexible when the width of the sidebar is adjusted.
- [ ] The right sidebar should also have the flexible grow property of the items (which was implemented for the toolbar).

![Draft-Projects-list-restructurations-2022-08-30-1059](https://user-images.githubusercontent.com/53972736/187399286-067b47c2-3981-459b-a88f-800daa5d545a.png)

**Comment by babsey** [comment]
We need to rethink where we can place the save icon to save the current project.

**Comment by babsey** [comment]
Current state of buttons bar:

![image](https://user-images.githubusercontent.com/620984/187937751-b32d2ccf-b1bd-419f-a219-94b351381911.png)
